src/main/java/skibidi/Parser.java

```diff
@@ -0,0 +1,42 @@
+package skibidi;
+
+import commands.*;
+
+/**
+ * Represents a parser to parse user input.
+ */
+public class Parser {
+    /**
+     * Parses the user input and returns the corresponding command.
+     *
+     * @param input The user input.
+     * @return The corresponding command.
+     * @throws SkibidiException If the user input is invalid.
+     */
+    public Command parse(String input) throws SkibidiException {
+
+        String normalizedInput = input.trim().toLowerCase();
+
+        if (normalizedInput.equals("bye")) {
```

Comment:
Can a switch statement or Enum be used to handle these if-else statements to make your code cleaner?